components/brave_wallet/browser/eth_tx_controller_unittest.cc

```diff
@@ -76,10 +76,13 @@ class TestEthTxControllerObserver
     tx_updated_ = true;
   }
 
-  void OnTransactionStatusChanged(mojom::TransactionInfoPtr tx) override {}
+  void OnTransactionStatusChanged(mojom::TransactionInfoPtr tx) override {
+    tx_changed_called_ = true;
```

Comment:
I think it's better to pass expected_status in constructor and check the status in the tx is as expected here.
Same as we did in ` OnUnapprovedTxUpdated`. 
And suggest renaming tx_changed_called_ to tx_status_changed_.

Reply:
done